### DISCUSSION TITLE
3월 9일 회의록

### DISCUSSION THREAD

**Discussion by zangsu** [zangsu/cs-test-service#2] [Announcements]
간단한 개발문화를 정해보았습니다. 오늘은 개발을 진행하기 위한 추가적인 과업들을 수행해야 합니다.
어떤 과업들을 수행해야 할까요?

**Comment by Moon-GD** [comment]
음... 오늘은 남은 시간을 생각해봤을 때
- [x] main repository에  main, dev branch 생성
- [x] frontEnd / backEnd 폴더 분기
- [x] .gitignore 파일 생성
- [x] 각자 로컬 환경에 clone ( 필요하다면 fork 후 clone )
- [x] 각자 개발 환경 구축
- [x] main repository 반영

**Reply by Moon-GD** [reply]
이렇게 해보는 게 어떨까요?

**Reply by zangsu** [reply]
확인했습니다. 우선 branch를 먼저 생성하는 것으로 시작하겠습니다!

**Comment by Moon-GD** [comment]
+ Github의 여러 요소(Git wiki, issue, Discussion, Project board) 활용
+ Framework의 간단한 활용

<br>

이번 프로젝트의 목적이 위와 같잖아요!
이런 사항들에 대해서 <strong><i>Github Wiki에 기술적 도전, 코드 프리징 날짜 선정, 최소 개발 가이드 라인 선정 </strong></i> 등을 정리해두면 좋을 것 같아요!

**Reply by zangsu** [reply]
네, 확인했습니다. 프로젝트 목표와 개발 일정 관련 회의를 진행하겠습니다.
